testing/simapp/sim_bench_test.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
 	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/testutil/sims"
```

Comment:
duplicate import here, its aliased below as `simtestutil`

Reply:
Got it!